### ISSUE TITLE
Add aria-current and hidden text to the step indicator component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#161]
### Problem

[molecules/molecules.step-indicator](https://a-ui.github.io/core_branding_scss/#molecules-step-indicator)

The status of the steps are only visually communicated. A screenreader user cannot know which steps are completed, which is current and which is incomplete.

```html
<ol class="m-step-indicator">
  <li class="is-completed"><span class="m-step-indicator__label">Step 1</span></li>
  <li class="is-completed"><span class="m-step-indicator__label">Step 2</span></li>
  <li class="is-active"><span class="m-step-indicator__label">Step 3</span></li>
  <li><span class="m-step-indicator__label">Step 4</span></li>
</ol>
```

![screenshot step indicator](https://user-images.githubusercontent.com/870049/98685116-44ee3e80-2367-11eb-8764-8a3ff49e5aa4.jpg)

### Solution

* Add hidden text 'completed' and 'incomplete'.
* Add aria-current="step" to the list item of the active step.

Suggestion:

```html
<ol class="m-step-indicator">
  <li class="is-completed"><span class="m-step-indicator__label">Step 1 <span class="u-screen-reader-only">completed</span></span></li>
  <li class="is-completed"><span class="m-step-indicator__label">Step 2 <span class="u-screen-reader-only">completed</span></span></li>
  <li aria-current="step" class="is-active"><span class="m-step-indicator__label">Step 3</span></li>
  <li><span class="m-step-indicator__label">Step 4 <span class="u-screen-reader-only">incomplete</span></span></li>
</ol>
```

### Articles and links

* [WAI-ARIA 1.2: aria-current (state)](https://www.w3.org/TR/wai-aria-1.2/#aria-current)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
